apollo-router/src/configuration/mod.rs

```diff
@@ -483,6 +496,7 @@ pub(crate) struct Supergraph {
     #[serde(default = "default_graphql_introspection")]
     pub(crate) introspection: bool,
 
+    /// Set to false to disable preview defer support
```

Comment:
I'll guess this is going to be a merge conflict against the recently merged graduation of preview support in #2378, but we'll want to remove the word `preview` here, too.

Reply:
Yeah, going to merge now.